### ISSUE TITLE
Research impact of disabling locationProvider's requireBase option

### ISSUE THREAD

**Issue by antmt** [QutBioacoustics/baw-client#170]
The base tag messes with the FuncIRI style attributed (e.g. `clip-path="url(#someThing)"`) making these resources essentially unfindable to the page.

This is detailed here: https://github.com/angular/angular.js/issues/8934

Solutions:
 - A set of directives for solving this problem: https://github.com/jeffbcross/angular-svg-base . Unclear whether these directives will work well with our dynamically rendered svg (my guess is the rendered svg would have to be transcluded)
- Specify an absolute reference to the clip-path. This is annoying but possible. Will be the current approach used by our dynamic svg (a d3 extension method / plugin will be made)
- Disable the requirement of the base tag in our app. Ideal solution. However, [this](https://github.com/angular/angular.js/commit/b3e09be58960b913fee3869bf36e7de3305bbe00) is part of angular 1.4 (I think?). Upgrading to angular 1.4 currently blocked.

**Comment by antmt** [comment]
Blocked by #171

**Comment by cofiem** [comment]
First point (set of directives) seems like it is worth trying before point 2.

**Comment by antmt** [comment]
1. is actually harder to do. 2. is what I've done.

**Comment by cofiem** [comment]
Ok, if you've already done 2., then fair enough.
